components/persistor/lib/knex/update.ts

```diff
@@ -250,4 +254,4 @@ module.exports = function (PersistObjectTemplate) {
                 obj[prop] = copyProps(obj[prop]);
         }
     }
-}
\ No newline at end of file
+};
```

Comment:
nit: typo? Don't think we need a semicolon at the end of class

Reply:
semicolon removed

Reply:
oh, actually this isn't a class definition - it's a module definition so gonna leave it